server/util.es6

```diff
@@ -142,4 +146,53 @@ util.getAllOpenApplications = (req, res) => {
     });
 };
 
+util.middleLayerAuth = () => {
+  let requestOptions = {
+    url: vcapServices.middleLayerBaseUrl + 'auth',
+    json: true,
+    body: { username: vcapServices.middleLayerUsername, password: vcapServices.middleLayerPassword }
```

Comment:
Once we move away from the mocks, this implementation will return and error. HTTP basic Auth, should be implemented as a header as follows
```
Build a string of the form username:password
Base64 encode the string
Supply an "Authorization" header with content "Basic " followed by the encoded string. For example, the string "fred:fred" encodes to "ZnJlZDpmcmVk" in base64, so you would make the request with the header as:
"Authorization: Basic ZnJlZDpmcmVk"
from https://developer.atlassian.com/jiradev/jira-apis/jira-rest-apis/jira-rest-api-tutorials/jira-rest-api-example-basic-authentication#JIRARESTAPIExample-BasicAuthentication-SupplyingBasicAuthheaders.
```

Alternatively, you can supply the user/pass in the url like so:
https://stackoverflow.com/a/18264755